.travis.yml

```diff
@@ -20,8 +20,9 @@ before_install:
 install:
   # - conda install --yes pip python=$TRAVIS_PYTHON_VERSION numpy scipy matplotlib ipywidgets
   - git clone https://github.com/simpeg/simpeg.git
-  - cd simpeg; git checkout dev; git pull; python setup.py build_ext --inplace; cd ..
+  - cd simpeg; git checkout dcip/dev; git pull; python setup.py build_ext --inplace; cd ..
```

Comment:
can we get this onto dev soon? it gets tricky if we start pointing too far up on SimPEG development. This is ok for now, but I will start an issue